src/Integrator/Integrator.cpp

```diff
@@ -52,7 +57,7 @@ Integrator::Integrator() : amrex::AmrCore()
         pp_query_default("regrid_int", regrid_int, 2);           // Regridding interval in step numbers
         pp_query_default("base_regrid_int", base_regrid_int, 0); // Regridding interval based on coarse level only
         pp_query_default("plot_int", plot_int, -1);               // Interval (in timesteps) between plotfiles (Default negative value will cause the plot interval to be ignored.)
-        pp_query_default("plot_dt", plot_dt, -1.0);                 // Interval (in simulation time) between plotfiles (Default negative value will cause the plot dt to be ignored.)
+        pp.query_default("plot_dt", plot_dt, "-1.0", Unit::Time());                 // Interval (in simulation time) between plotfiles (Default negative value will cause the plot dt to be ignored.)
```

Comment:
Is pp.query_default intentional? Or should it be pp_query_default?

Reply:
@vinagr we're slowly moving everything back to `pp.query...` For now, both are correct.